src/averageAges.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
+const getByKeyValue = (key, value) => people => {
+  return people.filter(person => person[key] === value);
+};
+
+const getMen = getByKeyValue('sex', 'm');
+const getWomen = getByKeyValue('sex', 'f');
+
+const getMothers = (people) => {
+  const women = getWomen(people);
+
+  return women
```

Comment:
```suggestion
  return people
```
Mothers are always women, so you can optimize logic.